doc/userguide/rules/ldap-keywords.rst

```diff
@@ -0,0 +1,44 @@
+LDAP Keywords
+=============
+
+.. role:: example-rule-action
+.. role:: example-rule-header
+.. role:: example-rule-options
+.. role:: example-rule-emphasis
+
+ldap.request.operation
+----------------------
+
+Suricata has a ``ldap.request.operation`` keyword that can be used in signatures to identify
+and filter network packets based on Lightweight Directory Access Protocol request operations.
+
+Syntax::
+
+ ldap.request.operation: operation;
```

Comment:
Looking good from a brief overview! Would like other opinions on the name of the keyword. This looks good to me but perhaps too long?

Reply:
For info, this name comes from json schema